lib/eventFactory.js

```diff
@@ -314,34 +314,43 @@ class EventFactory extends BaseFactory {
                         modelConfig.commit = commit;
                         modelConfig.createTime = (new Date()).toISOString();
 
+                        if (config.workflowGraph) {
```

Comment:
Note: this can be combined with the ` if (config.parentEventId)` later but having it here to be backward compatible. Since currently, having `parentEventId` doesn't mean it will have `workflowGraph` for sure.